src/chapters/sql-part1/studio.rst

```diff
@@ -0,0 +1,269 @@
+   #. `SQL Foreign Key <https://www.w3schools.com/sql/sql_foreignkey.asp>`__.
+
+   Follow the MySQL syntax instructions when there is a syntax variation
+   between the major databases.
+
+Setting up our two tables this way keeps our design *clean*. It puts data
+specific to directors in the ``directors`` table, places data specific to each
+movie in the ``movies`` table, and it provides a link between the two tables.
+
+Data such as the director's country is NOT an attribute of a movie, so there is
+no need to clutter a ``movie`` object or the ``movies`` table with that
+information. Instead, a country name is an attribute of the director, so that
+data should be stored in ``directors`` table.
+
+However, we might want to know that information for a given movie (e.g. we may
+want to find all the movies since 2010 that have French directors). So we need
+a way to filter the movies based on attributes from ``directors``. Solving this
+challenge is what relational databases are all about, and ``Foreign Keys``
+offer the solution.
+
+``INSERT`` Data From File
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Rather than adding entries line by line, you will use a prepared SQL file to
+speed up the process.
+
+#. Follow this link to the `movie-buff data <https://gist.github.com/jimflores5/5276e5cf15e19ae0923f809ee2367c7f>`__
+   repository.
+#. Click the *Download Zip* button to save a copy of the file to your machine.
+#. Double-click the zip file icon to extract the ``movie-buff-data.sql`` file
+   (it will be inside a folder).
+#. In MySQLWorkbench, click the *Open SQL* button, and select the
+   ``movie-buff-data.sql`` file.
+
+   .. figure:: ./figures/openSQLFileButton.png
+      :alt: Click "Open SQL" button.
+
+#. Double-click the ``movie-buff`` schema, then click the leftmost lightning
+   bolt icon to run the SQL script. This will populate the ``directors`` and
+   ``movies`` tables.
+
+   .. figure:: ./figures/runMovieBuffSql.png
+      :alt: Click the leftmost lightning bolt icon.
+      :scale: 80%
+
+#. Confirm that the tables contain data by clicking on the table icon next to
+   each name.
+
+   .. figure:: ./figures/moviesTableCheck.png
+      :alt: Select table contents button.
+
+OK! Your model is ready to accept queries.
+
+Your Assignment
+---------------
+
+For this studio, you'll practice writing SQL queries to perform various data
+retrieval and manipulation tasks. You will be using the ``movies`` and
+``directors`` tables described :ref:`above <directors-table>`, so if you still
+need to ``CREATE`` them, please do so now.
+
+Task List
+^^^^^^^^^
+
+Open up a new *Query* tab in MySQLWorkbench. In that tab, write the SQL
+commands to carry out each of the queries described below. As you complete each
+step, compare your SQL code and the output with a partner.
+
+#. List just the titles of all the movies in the database.
+
+#. List the title and year of each movie in the database in *DESCENDING* order
+   of the year released.
```

Comment:
They will probably need a hint on how to do this.

Reply:
Added.  Thanks.